test/util/test_ui_def.js

```diff
@@ -13,16 +13,21 @@ const _isUndefined = require('lodash/isUndefined')
 
 const VALID_ACTIONS = ['buy', 'sell', 'preview', 'submit']
 const VALID_FIELDS = [
+  'ui.checkbox_group',
```

Comment:
better to keep it in alphabetical order as before..
where is `ui.ticker', ui.tabs, reccuring_summary, reccuring_endDate` used ? and there is typo in recurring